draft/spec/index.html

```diff
@@ -797,6 +797,18 @@ <h2>Storage Hierarchies</h2>
 
     <section id="root-extensions">
         <h2>Extensions</h2>
+        <p>
+            The behavior of the storage root may be extended to support features from other specifications.  An OCFL
+            validator MUST ignore any files in the storage root it does not understand, but these files MUST not appear
```

Comment:
"these files" seems wrong as we are not talking about ones in the storage root. Also s/not/NOT/ in "MUST not".  How about:

> An OCFL validator MUST ignore any files in the storage root it does not understand. Additional files MUST NOT appear in other directories under the storage root.